spec/lib/proxmox_spec.rb

```diff
@@ -33,53 +33,55 @@
 
   describe 'get' do
```

Comment:
Could you change it with things look like "should launch a HTTP get request" to have a better description?